kgforge/specializations/models/rdf/service.py

```diff
@@ -272,3 +338,109 @@ def traverse_properties(properties) -> Tuple[Dict, Dict]:
         context.update({key: terms[key] for key in sorted(terms)})
 
         return {"@context": context} if len(context) > 0 else None
+
+    def transitive_load_shape_graph(
```

Comment:
Could you document it? For example, what is node_shape_uriref for

Reply:
Doc added